### DISCUSSION TITLE
[DatePicker] App crash during click on picker

### DISCUSSION THREAD

**Discussion by not-SAINT** [epam/UUI#1615] [Q&A] [answered]
Faced with a problem. When click on component to open picker and select date get an error:
![image](https://github.com/epam/UUI/assets/60464016/132a7bb7-d4b0-41d5-991b-0700d71bb5ea)

First get an error after update version to latest 5.1.2.
On version 5.1.0 on desktop all works good. With the same packages list.

Try to play with packages list, downgrade **dayjs** version from 1.11.5 to 1.10.7 (same as using UUI). It helps partially. 
Error rises on Android 13/Chrome.

I'm not sure about a bug on library side, but it looks strange. 
Any suggestions what it might be?

**Comment by AlekseyManetov** [comment, marked as answer]
Hi, some of our users encountered a similar and downgrade dayjs version helped for them. I am really confused about the root cause of this problem in the latest version, because we don't have any changes related dayjs for a long time.  Consequently, I'm unsure about why this error appeared.

Furthermore, I attempted to reproduce the error in a codesandbox environment, but didn't get any errors - https://codesandbox.io/s/uui-forked-4fnwxk?file=/Example.tsx
